### DISCUSSION TITLE
Popper inline style is missing

### DISCUSSION THREAD

**Discussion by treminaor** [floating-ui/floating-ui#2032] [Q&A] [answered]
I am using Popper in combination with Bootstrap 5.2 and am noticing that, when testing the tooltip several times in a row on the same element, a few times it will show the tooltip at the end of <body> instead of positioning it near the element that instantiated it. I've checked the element and noticed that when it appears at the end of <body>, its because the tooltip element has no inline styling applied to it! No positioning, no nothing. Just dumps an unstyled element at the end of the DOM.

I've tried every possible combination of different settings in the popperConfig to try and get the reference logic to work more consistently but nothing seems to affect this bug.

Anecdotally it seems to be worse when using "top" or "left" for position of tooltip as opposed to "bottom" or "right".

**Comment by atomiks** [comment, marked as answer]
Pretty sure this was a Bootstrap bug, I think they made a PR to fix it in a new version but I'm not sure. Not a bug with Popper, though.

**Reply by treminaor** [reply]
Thank you for bringing that to my attention - looks like they shipped a patch release less than a month ago to address this issue. I am on 5.2.2 and the patch release is 5.2.3.

**Reply by treminaor** [reply]
Yes, that fixed it! God, hours of my time wasted trying to work around that issue. Thought it was an issue with my page, not bootstrap!
